primers.go

```diff
@@ -103,3 +104,108 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
```

Comment:
Explain more about DNA barcoding. How does it work biologically? Where else besides nanopore stuff do you see it? Non-biologists won't understand much context here.